src/pages/Delivery/index.jsx

```diff
@@ -59,8 +59,8 @@ const Delivery = () => {
           }}
           actionText="Aceptar"
         >
-          <Text as="h1" color="primary" medium center>
-            El Dasher ha llegado
+          <Text as="h1" color="light-blue-11" medium center>
+            Hemos llegado
           </Text>
           <Text as="h4" color="gray" small center>
             Recoja su env&iacute;o
```

Comment:
Since is `as="h4"` I think the text is bolder, please check a way to change the font weight to normal to do the font look like the original design

![imagen](https://user-images.githubusercontent.com/66505715/158245465-c8c08277-51a6-481d-af67-25936366e7d1.png)